### DISCUSSION TITLE
iPad 使用 TikTok 是不是不需要添加 Rewrite 和 MitM ？

### DISCUSSION THREAD

**Discussion by iBenita** [Semporia/TikTok-Unlock#246] [Q&A] [answered]
在 iPad 开启飞航模式发现仅添加分流也可正常使用 TikTok !

**Comment by Semporia** [comment, marked as answer]
iPad 无线局域网机型是不需要 Rewrite 和 MitM 即可正常观看，所有功能都正常使用。
iPad 蜂窝网络机型如果有载入 SIM 卡的话，则需要搭配 Rewrite 和 MitM 才可以的；如没有载入 SIM 卡，则可以不用添加Rewrite 和 MitM。

**Comment by Ledrong** [comment]
你好，我是來自俄羅斯的學生。我想在https://buff.163.com/平台進行交易。但為此我需要驗證我的帳戶，而這只能使用支付寶和steam來完成。請您只需要驗證一下，一切都是安全的。我只需要一個個人資料
